opm/autodiff/StandardWells_impl.hpp

```diff
@@ -527,9 +532,11 @@ namespace Opm
 
         if ((*active_)[Oil] && (*active_)[Gas]) {
             // Incorporate RS/RV factors if both oil and gas active
-            const ADB& rv_perfcells = subset(state.rv, well_cells);
+            // the rv_perfcells could have been modified to accont for the effect of the solvent
+            // to remove dissolved oil and gas the pure rv value should be used.
+            const ADB& rv_perfcells_unmodified = subset(state.rv, well_cells);
             const ADB& rs_perfcells = subset(state.rs, well_cells);
```

Comment:
Modifying rs_perfcells is dangerous since I sometimes need the unmodified values.